docs/src/index.md

```diff
@@ -3,21 +3,50 @@
+Virtualization is one of the core enablers to transform the traditionally monolithic software stack within edge devices into isolated components with minimal TCB and clearly defined functionality.
+
+The Ghaf Platform utilizes a collection of virtual machines (VMs) to define a system.
+
+Contrary to the traditional monolithic OS, this concept allows to define and run host services in isolated environments, which breaks up the monolithic structure and allows for a modular system definition that is customizable for a specific use case. To this end, various applications and guest OSs can be deployed while simultaneously utilizing the Platform's features. 
+
+## Ghaf Platform Applications
+
+The Ghaf Platform development is focused on the transition to a modular architecture for edge devices. Products such as secure phones, drones, laptops, and other communication devices have unique challenges in their respective hardware and software ecosystems.
+
+Enabling the integration of individual technology stacks into an organizational framework can be a challenging task. The Ghaf Platform is designed to ease this process and enable research to overcome a number of challenges.
+
+## Design Principles
+
+The design principles influencing the architecture of the Ghaf Platform are the following:
+
+* Edge Security  
+  
+    The Ghaf security architecture under development by SSRC aims to provide an understandable yet comprehensive view of security controls in the Platform so that vendors can make informed decisions and adopt the Platform for their use cases. The security architecture and subsequent research will be published by SSRC in a series of technical white papers. 
+
+* Zero Trust
+
+    The Ghaf Platform aims to apply the general security principles of zero trust within the software architecture and to act as an enabler for ZTA for edge devices within organizations. 
+  
+* Trusted Computing Base 
 
-![Scope!](img/stack.drawio.png "Embedded Virtualization Scope")
+    The general principle for establishing the trusted Ghaf Platform code base is to rely on audited software and proven security modules while carefully evaluating and integrating new concepts. The modularized platform not only simplifies the integration of additional security measures but also facilitates the integration of hardware security features. Leveraging and contributing to open-source projects is not only a cornerstone for the Platform components' maintainability but also for the toolchain to increase transparency and auditability. By providing a hardened code base for the hypervisor and OS for the various VMs in the architecture, the Ghaf Platform leverages security benefits across all modules.
 
```

Comment:
I think we could mention here as design principles:
* Configurable, Declarative and Reproducible 
If you agree, we can break it down together. These principles are key design drivers for our technology choices and architecture break down.

Reply:
I can just add a new point to the list, and we can add more details in some other future pull request.

Reply:
Works for me.